go.mod

```diff
@@ -4,10 +4,10 @@ go 1.18
 
 require (
 	github.com/StackExchange/wmi v1.2.1
-	github.com/ghodss/yaml v1.0.0
 	github.com/jaypipes/pcidb v1.0.0
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v0.0.3
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
How about `gopkg.in/yaml.v3` instead? :)

Reply:
Updated to `gopkg.in/yaml.v3`